api/app/tests/apps/signals/test_workflow.py

```diff
@@ -64,3 +64,58 @@ def test_reopen_example_fails(self, patched_udate_status_signal):
         self.assertEqual(self.signal.status.state, workflow.BEHANDELING)
         self.assertEqual(patched_udate_status_signal.send_robust.call_count, 4)
         self.assertEqual(Status.objects.filter(state=workflow.HEROPEND).count(), 0)
+
+
+class TestTransistion(TransactionTestCase):
+    @mock.patch('signals.apps.signals.managers.update_status', autospec=True)
+    def test_to_allowed_states(self, patched_update_status_signal):
+        call_counter = 0
+
+        for current_state in workflow.ALLOWED_STATUS_CHANGES:
+            allowed_states = workflow.ALLOWED_STATUS_CHANGES[current_state]
+            for allowed_state in allowed_states:
+                signal = factories.SignalFactory.create(status__state=current_state)
+
+                new_status_data = {'state': allowed_state,
+                                   'text': 'Dit is een test.'}
+
+                if allowed_state == workflow.TE_VERZENDEN:
+                    new_status_data.update({'target_api': Status.TARGET_API_SIGMAX})
+
+                Signal.actions.update_status(new_status_data, signal)
+
+                self.assertEqual(signal.status.state, allowed_state)
+
+                call_counter += 1
+
+        self.assertEqual(patched_update_status_signal.send_robust.call_count, call_counter)
+
+    @mock.patch('signals.apps.signals.managers.update_status', autospec=True)
+    def test_not_allowed_states(self, patched_update_status_signal):
```

Comment:
docstring, maybe?

Reply:
Added a simple docstring explaining the tests